tools/database/wowhead_tooltips.go

```diff
@@ -697,6 +688,7 @@ var dodgeGemStatRegexes = []*regexp.Regexp{regexp.MustCompile(`\+([0-9]+) Dodge
 var parryGemStatRegexes = []*regexp.Regexp{regexp.MustCompile(`\+([0-9]+) Parry Rating`)}
 var resilienceGemStatRegexes = []*regexp.Regexp{regexp.MustCompile(`\+([0-9]+) Resilience Rating`)}
 var allResistGemStatRegexes = []*regexp.Regexp{regexp.MustCompile(`\+([0-9]+) Resist All`)}
+var masteryGemStatRegexes = []*regexp.Regexp{regexp.MustCompile(`\+([0-9]+) Mastery Rating`)} //https://www.wowhead.com/cata/spell=101748/zen-elven-peridot
```

Comment:
same as above - Rating vs no Rating in the stat wording

Reply:
Yep. Check https://www.wowhead.com/cata/item=65208/lightning-charged-spaulders and https://www.wowhead.com/cata/spell=101748/zen-elven-peridot
 in socket and gem bonus it is worded as "Mastery Rating"